build/tools/postman-avalanche-collection.md

```diff
@@ -0,0 +1,85 @@
+# Avalanche Postman Collection
+
+## What is Postman?
+
+Postman is a free tool used by developers to quickly and easily send REST, POST, and GraphQL requests and test APIs. It is available as both an online tool and an application for Linux, MacOS and Windows. Postman allows you to quickly issue API calls and see the responses in a nicely formatted, searchable form.
+
+We have made a Postman collection for [Avalanche](https://docs.avax.network), that includes all the public API calls that are available on [AvalancheGo instance](../release-notes/avalanchego.md), allowing you to quickly issue commands to your node and see the response, without having to copy and paste long and complicated `curl` commands.
+
+Along with the API collection, there is also the example Avalanche environment for Postman, that defines common variables such as IP address of the node, your Avalanche addresses and similar common elements of the queries, so you don't have to enter them multiple times.
+
+Combined, they will allow you to easily keep tabs on your node, check on its state and do quick queries to find out details about its operation.
+
+## Setup
+
+### Postman installation
+
+Postman can be installed locally or used as a web app. We recommend installing the application, as it simplifies operation. You can download Postman from its [website](https://www.postman.com/downloads/). It is recommended that you sign up using your email address as then your workspace can be easily backed up and shared between web app and the app installed on your computer.
+
+![Download Postman](../../.github/postman_01_download.png)
+
+After you installed the application, run it. It will prompt you to create an account or log in. Do so. Again, it is not necessary, but recommended.
+
+### Collection import
+
+Select `New workspace` from Workspaces tab and follow the prompts to crate a new workspace. This will where the rest of the work will be done.
+
+![New workspace](../../.github/postman_02_workspace.png)
+
+We're ready to import the collection. On the header of the Worskspaces tab select `New` and switch to `Link` tab.
+
+![Import collection](../../.github/postman_03_import.png)
+
+There, in the URL input field paste the link to the collection:
+
+```
+https://raw.githubusercontent.com/ava-labs/avalanche-docs/master/scripts/Avalanche.postman_collection.json
```

Comment:
I'd prefer to keep the postman collection in it's own repo for now. Please update this URL to reference `https://github.com/ava-labs/avalanche-postman-collection`. TY